ts/lib/auth/computeclient.ts

```diff
@@ -102,7 +116,7 @@ export default class Compute extends Auth2Client {
           err.message = helpfulMessage;
         } else {
           err = new Error(helpfulMessage);
```

Comment:
Eugh, sadly it gets super hairy.  These errors are threaded through tons of callbacks, some of which originate outside of here (in request).  This seemed like the safest way to pin the code on the error.